docs/spark-queries.md

```diff
@@ -95,21 +95,39 @@ The above list is in order of priority. For example: a matching catalog will tak
 
 #### SQL
 
-Spark 3.3 and later supports time travel in SQL queries using `TIMESTAMP AS OF` or `VERSION AS OF` clauses
+Spark 3.3 and later supports time travel in SQL queries using `TIMESTAMP AS OF` or `VERSION AS OF` clauses.
+The `VERSION AS OF` clause can be used for time traveling to a specific snapshot via the following options:
```

Comment:
the wording seems a bit overly complex. What about we talk about `VERSION AS OF` with the data type? If it is long, then it is snapshot ID. If it is string, then it follows what is described below.

Reply:
Sure this seems reasonable and should be clear for readers